access reviews to v1 - using doc stub generator (#12207)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/accessreviewinstance-applydecisions.md

---
title: "accessReviewInstance: applyDecisions"
description: "Apply decisions on an accessReviewInstance."
author: "isabelleatmsft"
localization_priority: Normal
ms.prod: "governance"
doc_type: apiPageType
---

# accessReviewInstance: applyDecisions
Namespace: microsoft.graph

Apply review decisions to the resource reviewed in an [accessReviewInstance](../resources/accessreviewinstance.md).

> Note that decisions will be applied automatically if the `autoApplyDecisionsEnabled` parameter is `True` in the review's [accessReviewScheduleSettings](../resources/accessreviewschedulesettings.md).

Reviewer: ```suggestion
Decisions are applied automatically if the `autoApplyDecisionsEnabled` parameter is `true` in [accessReviewScheduleSettings](../resources/accessreviewschedulesettings.md) of the review.
```

## Permissions
One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

|Permission type|Permissions (from least to most privileged)|
|:---|:---|
|Delegated (work or school account)|AccessReview.ReadWrite.All|
|Delegated (personal Microsoft account)|Not supported.|
|Application|AccessReview.ReadWrite.All|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
POST /identityGovernance/accessReviews/definitions/{accessReviewScheduleDefinitionId}/instances/{accessReviewInstanceId}/applyDecisions
```

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|

## Request body
Do not supply a request body for this method.

## Response

If successful, this action returns a `204 No Content` response code.

## Examples

### Request
<!-- {
  "blockType": "request",
  "name": "accessreviewinstance_applydecisions"
}
-->
``` http

> POST https://graph.microsoft.com/v1.0/identityGovernance/accessReviews/definitions/{accessReviewScheduleDefinitionId}/instances/{accessReviewInstanceId}/applyDecisions

Reviewer: Real looking id values.
